policies/optimized-images.md

```diff
@@ -0,0 +1,342 @@
+# Optimized Image Policies Explainer
+
+loonybear@, last updated: 4/25/2018
+
+<span style="color:#38761d;">Status: Draft Proposal</span>
+
+
+## Goal
+
+Images make up majority of the downloaded bytes on websites. In addition, images often occupy a significant amount of visual space. Optimizing images can improve loading performance and reduce network traffic. Surprisingly, more than half of the sites on the web, including advanced ones, are shipping unoptimized images. This means these sites can all achieve a performance improvement by serving optimized images.
+
+Optimized image policies are aiming to solve problems with sites shipping images that are poorly compressed or unnecessarily large.
+
+
+## What are "Optimized Image Policies"?
+
+Optimized image policies introduce a set of restrictions (policies) on images that can be applied with dev-time enforcement. An image will be rendered with **inverted colors** when violating a policy, making it easy for web developers to identify and fix the error.
+
+
+### Optimized image policies
+
+*   **["legacy-image-formats" policy](#legacy-image-formats)**
+    *   Images must be of one of the _**modern formats***_ (JPEG, PNG, WEBP, etc).
+*   **["maximum-downscaling-image" policy](#maximum-downscaling-image)**
+    *   Images must not be bigger than its container size by more than _**X times***_ .
```

Comment:
How do we reason about device DPR? Many sites today provide "2x" resources in <img>.. based on assumption that most mobile devices are at least 2x.